changelogs/fragments/3237-copr-fix_chroot_naming.yml

```
@@ -0,0 +1,3 @@
+bugfixes:
+  - copr - fix chroot naming issues, centos-stream changed naming to centos-stream-<number> (e.g. centos-stream-8).
+  - copr - fix corresponding integration tests.
```

Test changes don't need a changelog fragment. For the other entry, here's a suggestion with better formatting:
```suggestion
  - copr - fix chroot naming issues, ``centos-stream`` changed naming to ``centos-stream-<number>`` (for exmaple ``centos-stream-8``) (https://github.com/ansible-collections/community.general/pull/3237).
```